tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+
+import pytest
+
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutor
+from tests.common.schedulers_common import get_scheduler_commands
+from tests.storage.storage_common import verify_directory_correctly_shared
+
+
+# For EFS tests, only use regions defined in AVAILABILITY_ZONE_OVERRIDES in conftest
```

Comment:
we could also create an ad hoc compute subnet that is in a different AZ (for this test only) rather than relying on the overrides